onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -74,9 +93,11 @@ def optimize_for_ort(model: ir.Model, config_name: str | None = None) -> ir.Mode
             If None, the default configuration will be used.
 
     Returns:
-        The optimized model.
+        A tuple containing:
+        - The optimized `ir.Model` after applying transformer-specific fusions.
+        - A dictionary with a count of each of the fusions applied.
     """
 
-    fuse_xformers(model)
+    model, fusion_count = fuse_xformers(model)
     rewrite(model, ORT_PATTERN_REWRITE_RULES)
-    return model
+    return model, fusion_count
```

Comment:
This is fine for now. (Later it might be worth extending fusion_count to return stats from ORT_PATTERN_REWRITE_RULES too, but that requires updating the pattern-matcher to return this info, so better to do it separately.

Reply:
Made a note of this, will do this as part of a separate PR.